auth/server/src/main/java/com/seagates3/authserver/AuthServerConfig.java

```diff
@@ -131,19 +133,43 @@ public static void logConfigProps() {
      * @throws GeneralSecurityException
      */
     public static void loadCredentials() throws GeneralSecurityException, Exception {
-
-        Properties authServerConfig = AuthServerConfig.authServerConfig;
-        String encryptedPasswd = authServerConfig.getProperty("ldapLoginPW");
-        Path keyStoreFilePath = getKeyStorePath();
-        PrivateKey privateKey = JKSUtil.getPrivateKeyFromJKS(
-                                keyStoreFilePath.toString(), getCertAlias(),
-                                getKeyStorePassword());
-        if (privateKey == null) {
-             throw new GeneralSecurityException("Failed to find Private Key ["
-                + keyStoreFilePath + "].");
-        }
-        ldapPasswd = RSAEncryptDecryptUtil.decrypt(encryptedPasswd,
-                     privateKey);
+       logger = LoggerFactory.getLogger(AuthServerConfig.class.getName());
+       String keystorePasswd = null;
+       String encryptedPasswd = authServerConfig.getProperty("ldapLoginPW");
+       Path keyStoreFilePath = getKeyStorePath();
+       try {
+         String cmd = authServerConfig.getProperty("cortexSecCmd");
+         logger.info("Command is : " + cmd);
+         Process cortxsec = Runtime.getRuntime().exec(cmd);
+         int exitVal = cortxsec.waitFor();
+         if (exitVal != 0) {
+           throw new IOException("Process exited with error code");
```

Comment:
Does caller handle this exception, please check all places if exception handled by caller in all places